exporter/awsemfexporter/emf_exporter.go

```diff
@@ -106,10 +106,15 @@ func (emf *emfExporter) pushMetricsData(_ context.Context, md pmetric.Metrics) e
 
 	for _, groupedMetric := range groupedMetrics {
 		cWMetric := translateGroupedMetricToCWMetric(groupedMetric, emf.config)
-		putLogEvent := translateCWMetricToEMF(cWMetric, emf.config)
+		putLogEvent, err := translateCWMetricToEMF(cWMetric, emf.config)
+		if err != nil {
+			return err
+		}
 		// Currently we only support two options for "OutputDestination".
 		if strings.EqualFold(outputDestination, outputDestinationStdout) {
-			fmt.Println(*putLogEvent.InputLogEvent.Message)
+			if putLogEvent.InputLogEvent.Message != nil {
```

Comment:
Can `putLogEvent.InputLogEvent` be nil?

Reply:
Thank you, good catch. I added two additional nil checks for the parent objects.